src/Boot/BootstrapManager.php

```diff
@@ -410,15 +390,12 @@ protected function maxPhaseLimit($bootstrap_str)
     /**
      * Bootstrap to the specified phase.
      *
-     * @param int $max_phase_index
```

Comment:
Shouldn’t we only remove the type hint and not the whole line? The description lacks context without the `@param` line above.

Reply:
I'll see if there's a way to teach rector to leave those alone.

Reply:
I found a way to make this stop, with a couple of one-line hacks to Rector.